tensorflow/core/debug/debug_io_utils.cc

```diff
@@ -147,6 +146,7 @@ Status DebugIO::PublishDebugMetadata(
   Status status;
   for (const string& url : debug_urls) {
     if (str_util::Lowercase(url).find(kGrpcURLScheme) == 0) {
+#if defined(PLATFORM_GOOGLE)
```

Comment:
Doesn't this break existing open-source code, or was this path inaccessible before?